ci-operator/step-registry/baremetal/lab/pre/serial-console/baremetal-lab-pre-serial-console-commands.sh

```
@@ -18,20 +18,38 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
     bmc_pass=$(echo "$bmhost" | jq -r '.bmc_pass')
     bmc_address=$(echo "$bmhost" | jq -r '.bmc_address')
 
-    bios_attributes=$(curl -k -u "$bmc_user:$bmc_pass" https://$bmc_address/redfish/v1/Systems/System.Embedded.1/Bios | yq .Attributes)
-    model=$(curl -k -u "$bmc_user:$bmc_pass" https://$bmc_address/redfish/v1/Systems/System.Embedded.1 | yq .Model)
+    bios_attributes=$(curl --fail --silent --show-error -k -u "$bmc_user:$bmc_pass" https://$bmc_address/redfish/v1/Systems/System.Embedded.1/Bios | yq .Attributes)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)